src/main/java/com/me4502/modularframework/module/ModuleWrapper.java

```diff
@@ -134,6 +139,16 @@ public Object getModule() throws ModuleNotInstantiatedException {
     //Cache the annotation. I have no idea what the performance overhead for not doing this is, but meh.
     private Module annotation;
 
+    public String getId() {
+        try {
+            return getAnnotation().moduleId();
+        } catch (ClassNotFoundException e) {
+            e.printStackTrace();
+        }
+
+        return moduleClassName.toLowerCase();
```

Comment:
Once default is done, this won't be needed. It'll return null in exceptional circumstances.